data/scripts/creaturescripts/customs/freequests.lua

```diff
@@ -263,7 +265,7 @@ local questTable = {
 	{storage = Storage.FriendsandTraders.TheBlessedStake, storageValue = 12},
 	{storage = 100157, storageValue = 1},
 	{storage = 2500, storageValue = 0},
-	{storage = Storage.WrathoftheEmperor.Questline, storageValue = 1},
+	{storage = Storage.WrathoftheEmperor.Questline, storageValue = 29,
```

Comment:
Need to add missing "}"
```suggestion
	{storage = Storage.WrathoftheEmperor.Questline, storageValue = 29},
```